prime-angular-frontend/nginx.template.conf

```diff
@@ -36,6 +36,7 @@ server {
     }
     location /api/docman/ {
         proxy_pass        http://document-manager$SUFFIX-backend:6001/;
+        client_max_body_size  20m;
```

Comment:
Why was 20 MB chosen here? The document manager has a maximum file size of 400 MB:
![image](https://user-images.githubusercontent.com/39168456/120246940-9e945880-c226-11eb-8ffc-22f9f821a51c.png)

Reply:
While the frontend restricts uploads to 3MB:
![image](https://user-images.githubusercontent.com/39168456/120247025-f763f100-c226-11eb-8eaa-9c1b2dee69fb.png)

Reply:
Someone was saying that we may need to deal with 20 MB business licence files so I was just testing that it is possible

Reply:
Ah, I see. In that case it should probably be set to the maximum size the Document Manager can accept, 400MB.

Reply:
I've removed this change since it seems the scanned business licence files are not expected to be larger than 1 MB